cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/DefaultPreviewRunnerModule.java

```diff
@@ -175,6 +178,9 @@ protected void configure() {
     bind(PreviewStore.class).to(DefaultPreviewStore.class).in(Scopes.SINGLETON);
     bind(Scheduler.class).to(NoOpScheduler.class);
 
+    bind(PreviewRequestQueue.class).to(DefaultPreviewRequestQueue.class).in(Scopes.SINGLETON);
+    expose(PreviewRequestQueue.class);
```

Comment:
Why expose?

Reply:
PreviewManager gets the instance of this class using preview injector and adds request to it, which is why its exposed.

Reply:
Shouldn't the queue belongs to the Preview Manager internally? Why the binding is in separate injector?

Reply:
Refactoored the guice bindings in the last commit. PTAL. thanks!